frontend/src/components/Forms/CoffeeChatsForm/CoffeeChatsDashboard.tsx

```diff
@@ -226,34 +226,44 @@ const RejectedChatsDisplay = ({ coffeeChats }: { coffeeChats: CoffeeChat[] }) =>
   </div>
 );
 
-const ArchivedChatsDisplay = ({ coffeeChats }: { coffeeChats: CoffeeChat[] }) => (
-  <div className={styles.archived_display}>
-    <Table celled style={{ border: '0.5px solid black' }}>
-      <Table.Header>
-        <Table.Row>
-          <Table.HeaderCell>Chat Details</Table.HeaderCell>
-        </Table.Row>
-      </Table.Header>
-      <Table.Body>
-        {coffeeChats.map((chat) => (
-          <Table.Row key={chat.uuid}>
-            <Table.Cell>
-              <div>
-                Coffee Chat with {chat.otherMember.firstName} {chat.otherMember.lastName} (
-                {chat.otherMember.netid})
-              </div>
-              <div>Category: {chat.category}</div>
-              <div>
-                <a href={chat.slackLink} target="_blank" rel="noopener noreferrer">
-                  Slack Link
-                </a>
-              </div>
-            </Table.Cell>
+const ArchivedChatsDisplay = ({ coffeeChats }: { coffeeChats: CoffeeChat[] }) => {
+  const validChats = coffeeChats
+    .filter((chat) => chat.otherMember)
+    .filter((chat) => chat.status === 'approved');
+
+  return (
+    <div className={styles.archived_display}>
+      <Table celled style={{ border: '0.5px solid black' }}>
+        <Table.Header>
+          <Table.Row>
+            <Table.HeaderCell>Chat Details</Table.HeaderCell>
           </Table.Row>
-        ))}
-      </Table.Body>
-    </Table>
-  </div>
-);
+        </Table.Header>
+        <Table.Body>
+          {validChats.length === 0 ? (
```

Comment:
```suggestion
          {!validChats.length ? (
```

Also I was wondering if we wanted to place the `ptag` in the `Table.Row` tags as well! Other than that it looks great!